static/include/services/apis/generated/vision.md

```diff
@@ -615,6 +615,52 @@ For more information, see the [Flutter SDK Docs](https://flutter.viam.dev/viam_s
 {{% /tab %}}
 {{< /tabs >}}
 
+### GetProperties
+
+Fetch information about what vision methods the vision service provides.
```

Comment:
I think this description is a bit confusing as it could be understood to mean find out which methods the vision service in general provides. What do you think about this or similar?
```suggestion
Fetch information about which vision methods a given vision service supports.
```